src/openms/source/ANALYSIS/MAPMATCHING/QTClusterFinder.cpp

```diff
@@ -513,6 +640,11 @@ void QTClusterFinder::createConsensusFeature_(ConsensusFeature& feature,
               {
                 continue; // conditions not satisfied
               }
+              if (use_IDs_ && neighbor_feature->getAnnotations().empty())
```

Comment:
add comment what is checked here. e.g., // if IDs are used during linking check if unidentified features are too far off regarding RT